create_index.py

```diff
@@ -12,7 +12,7 @@
 @app.cli.command()
 def create_index():
     """Initialize the search index."""
-    index.SearchSession().create_index()
+    index.SearchSession.create_index()
```

Comment:
```suggestion
    index.SearchSession.current_session().create_index()
```

Reply:
doesn't this remove the whole point of having a metaclass on `index.SearchSession`? if `index.SearchSession.create_index()` is the same as with the syntax sugar, why use the more verbose?

Reply:
fair enough

Reply:
One thing being explicit does help is reduce the number of pylint: no-value-for-paramter errors:

> No value for argument 'query' in unbound method call.

Reply:
yeah, it also helps with mypy. tbh I kind of regret the metaclass thing, a bit tempted to roll it back. but that shouldn't stop you here